### ISSUE TITLE
Frame step for "Bake to Keymesh" operator

### ISSUE THREAD

**Issue by Dereuz24** [nickberckley/keymesh#38]
**Describe the feature you'd like to see**

Now that's possible to bake down an animation from an object with an armature modifier, it would be necessary to have a setting in the operator to select the frame step instead of baking the entire range in 1s

**Comment by nickberckley** [comment]
That's easy to support, but I didn't think it would be important. Baking is supposed to look exactly the same usually, and retain whatever was given. I assume that if user wants stepped animation first they'll make armature animation stepped and then bake, because otherwise you don't really know what you'll get after the bake.

**Comment by Dereuz24** [comment]
Yes exactly, this would be for situations like a stepped animation across 100 frames, if it's a regular interval (example 1 frame every 10 frames) it would save a lot of time spent deleting keymesh objects before sculpting or modifying them

This manual process would be necessary in situations with irregular stepped animation but for long regular intervals it would save a lot of time

**Comment by nickberckley** [comment]
I guess it would improve performance as well if only 50% of the range is iterated, so it's worth doing. But you give good example, if user already has animation on 2s bake would return on 1s. I didn't think of that.
